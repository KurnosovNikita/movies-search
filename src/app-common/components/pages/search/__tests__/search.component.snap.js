import React from 'react';
import {shallow} from 'enzyme';
import Search from '../search.component';
import configureMockStore from "redux-mock-store";
import {Provider} from "react-redux";

const mockStore = configureMockStore();
const store = mockStore({});

describe('Search component', () => {
    let state;

    beforeEach(() => {
        state = {
            movies: [],
            searchFieldValue: '',
            activeFilter: 'title',
            activeSortByLink: 'rating',
        };
    });

    it('should render <Search> correctly', () => {
        const component = shallow(
          <Provider store={store}>
            <Search/>
          </Provider>);

        expect(component).toMatchSnapshot();
    });
});
