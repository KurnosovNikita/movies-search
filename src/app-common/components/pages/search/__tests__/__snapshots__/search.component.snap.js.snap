// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render <Search> correctly 1`] = `
<Fragment>
  <Header
    activeFilter="title"
    handleFieldInput={[Function]}
    handleFieldKeyDown={[Function]}
    handleFilterChange={[Function]}
    handleSearchClick={[Function]}
  />
  <Main
    activeSortByLink="rating"
    handleImageClick={[Function]}
    handleSortLinkClick={[Function]}
    movies={Array []}
  />
</Fragment>
`;
